docs/sources/flow/reference/components/discovery.kubernetes.md

```diff
@@ -269,7 +269,7 @@ omitted, all namespaces are searched.
 Name | Type | Description | Default | Required
 ---- | ---- | ----------- | ------- | --------
 `own_namespace` | `bool`   | Include the namespace the agent is running in. | | no
-`names` | `list(string)` | List of namespaces to search. | | no
+`names` | `[]string` | List of namespaces to search. | | no
```

Comment:
Did we want to revert this change as well?

Suggestion in case you want to apply from GH:
```suggestion
`names` | `list(string)` | List of namespaces to search. | | no
```